types/store/shapediverStoreSession.ts

```diff
@@ -51,6 +62,18 @@ export interface IShapeDiverStoreSession {
 	createSession: (
 		dto: SessionCreateDto,
 		callbacks?: IShapeDiverStoreSessionCallbacks,
+		load?: boolean,
+	) => Promise<ISessionApi | undefined>;
```

Comment:
@MajorMeerkatThe3rd let's document the `load` parameter, and how it is used in combination with `pendingSessions` and `createPendingSession`